### DISCUSSION TITLE
How do I convert models from q4_2?

### DISCUSSION THREAD

**Discussion by skidd-level-100** [ggml-org/llama.cpp#1399] [Q&A]
In light of the current readme stating:
warning TEMPORARY NOTICE ABOUT UPCOMING BREAKING CHANGE warning

The quantization formats will soon be updated: https://github.com/ggerganov/llama.cpp/pull/1305

All ggml model files using the old format will not work with the latest llama.cpp code after that change is merged
:
I would like to have a few command examples for when this change hits to convert my alpaca-enhanced model (https://huggingface.co/Pi3141/alpaca-7b-native-enhanced) to whatever the new format will be, also any other conversion commands
are great as I have a couple more models I like (Wizard-7B-uncesored, and more). 

https://github.com/ggerganov/llama.cpp/discussions/1288
is empty ATM

**Comment by Green-Sky** [comment]
you could get the originals https://huggingface.co/8bit-coder/alpaca-7b-nativeEnhanced/ and convert them yourself. (that is all Pi3141 did in this case)

or just wait for ppl to roll out the new files.

**Reply by Green-Sky** [reply]
however, this model is relatively old. use something that is newer, like vicuna or wizardml etc..

**Reply by skidd-level-100** [reply]
> you could get the originals https://huggingface.co/8bit-coder/alpaca-7b-nativeEnhanced/ and convert them yourself. (that is all Pi3141 did in this case)
> 
> or just wait for ppl to roll out the new files.

thanks for this but could you give me some command syntax --help does not help me and I cant seem to find enough online

**Reply by Green-Sky** [reply]
this section in the readme should get you going :)
https://github.com/ggerganov/llama.cpp#prepare-data--run
